include/sys/dmu_send.h

```diff
@@ -76,4 +76,63 @@ int dmu_recv_stream(dmu_recv_cookie_t *drc, struct vnode *vp, offset_t *voffp,
 int dmu_recv_end(dmu_recv_cookie_t *drc, void *owner);
 boolean_t dmu_objset_is_receiving(objset_t *os);
 
+#define	dprintf_drro(type, drro, err) do { \
+	if (err != 0) \
+	dprintf("drr_type = %u obj = %llu type = %u bonustype = %u " \
+	"blksz = %u bonuslen = %u cksumtype = %u compress = %u " \
```

Comment:
I think in other cases we indent the macro body properly (i.e. the `dprintf` would be 2 tabs, and subsequent continuation lines 2 tabs + 4 spaces).